test/test.js

```diff
@@ -33,6 +33,13 @@ describe('chai-immutable', function () {
     });
 
     describe('equal method', function () {
+      it(
+        'fails when only the "expected" value is an Immutable collection',
+        function () {
+          assert.throws(expect([]).to.equal.bind(null, List()));
```

Comment:
Kudos on providing failing tests beforehand :-)

A few things:

- Why the `bind` method here?
- When testing that something throws an exception, would you mind [using this form](http://chaijs.com/api/bdd/#throw) instead?
- This test passes with and without your fix, are you sure it really tests what you wanted to test?

That being said, the entire test suite checks that the expected behaviors but never that the proper exceptions are thrown, which is quite pointless in the end... I should work on that, thanks for reminding me :-)